Functions/InputOutput/xASL_io_DcmtkRead.m

```diff
@@ -16,21 +17,32 @@
 %               This function also corrects formating of certain parameters.
 %
 % -----------------------------------------------------------------------------------------------------------------------------------------------------
-% EXAMPLE:      ...
+% EXAMPLE:      header = xASL_io_DcmtkRead('/tmp/file.dcm', [], 1);
 %
 % -----------------------------------------------------------------------------------------------------------------------------------------------------
 % REFERENCES:
 % __________________________________
-% Copyright 2015-2023 ExploreASL
+% Copyright 2015-2024 ExploreASL
 
-if nargin == 1
-	bPixel = 0;
+if nargin < 2 || isempty(bPixel)
+	bPixel = false;
 end
 
-try
-	% Read using DCMTK
-	header = xASL_mex_DcmtkRead(filepath, bPixel);
-catch
+if nargin < 3 || isempty(bTryDCMTK)
+	bTryDCMTK = true;
+end
+
+if bTryDCMTK
+	try
+		% Read using DCMTK
+		header = xASL_mex_DcmtkRead(filepath, bPixel);
+	catch ME
+        fprintf('%s%s\n','Warning, xASL_io_DcmtkRead failed with following warning: ', ME.message);
```

Comment:
Better to issue a Matlab warning and put ME.message on the next line right?

Reply:
Actually - I put a fprintf on purpose. This is about DCMTK failing and a fallback to SPM reading of DICOM. So things work fine in the end. That's why I think we can print a minimal warning rather than using 'warning' as this would print 10 lines per each file for a problem that is moderately severe.

I have modified the warning to make this clear.